### ISSUE TITLE
Move email off google

### ISSUE THREAD

**Issue by iancrowther** [nodeconf/collective#12]
Nodeschool is set up so that each chapter can have its own email address.  It would be great to mirror this functionality.

@brianloveswords can you advise here?

**Comment by brianloveswords** [comment]
Yep, check out this repo: https://github.com/nodeschool/email – which itself was forked (and slightly modified) from https://github.com/nodejs/email. I also setup a webhook listener so it would auto-run whenever any PR lands.
